examples/tts/conf/rad-tts_feature_pred.yaml

```diff
@@ -0,0 +1,339 @@
+name: RadTTS
+sample_rate: 22050
+
+train_dataset: ???
+validation_datasets: ???
+ckpt_path: ???
+export_dir: ???
+sup_data_path: ???
+sup_data_types: ["log_mel", "align_prior_matrix", "pitch", "voiced_mask", "p_voiced", "energy"] 
+
+whitelist_path: "nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv"
+
+# LJSpeech stats (per frame), train
+pitch_mean: 212.35873413085938
+pitch_std: 68.52806091308594
+
+# default values from librosa.pyin
+pitch_fmin: 65.40639132514966
+pitch_fmax: 2093.004522404789
+
+# default values for sample_rate=22050
+n_mels: 80
+n_window_size: 1024
+n_window_stride: 256
+n_fft: 1024
+lowfreq: 0
+highfreq: 8000
+window: "hann"
+
+pitch_loss_scale: 0.1
+durs_loss_scale: 0.1
+mel_loss_scale: 1.0
+
+phoneme_dict_path: "scripts/tts_dataset_files/cmudict-0.7b_nv22.07"
+heteronyms_path: "scripts/tts_dataset_files/heteronyms-030921" #"scripts/tts_dataset_files/heteronyms-030921"
+adlr_symbol_id_mapper_path: "scripts/tts_dataset_files/mapping.txt"
+
+model:
+  target: nemo.collections.tts.models.RadTTSModel
```

Comment:
Is this target used somewhere in the recipe? If not, pls remove it.

Reply:
This is for importing the RadTTS model for inference from a NeMo file